config/settings.yml

```diff
@@ -494,7 +494,7 @@ bgs:
   ssl_verify_mode: 'peer'
 
 va_mobile:
-  url: 'https://veteran.apps.va.gov/'
+  url: 'https://veteran.apps.va.gov'
```

Comment:
@saneshark this is the trailing slash to causes local specs and the base path in fhir configuration out of sync. In staging and production the foward proxy does not have a trailing slash. The v0 VOAS services as they use this base url (not a base path, ie `https://veteran.apps.va.gov` vs `https://veteran.apps.va.gov/vsp/v1/`) don't suffer from the issue as Faraday strips extra slashes immediately after the domain name.